badge_store/main.py

```diff
@@ -43,51 +44,81 @@ def show_apps(c):
     clear()
     menu_items = [{"title": a, "app": a} for a in store.get_apps(c)]
 
-    option = prompt_option(menu_items, none_text="Back", title=title)
+    option = prompt_option(menu_items, none_text="Back", title="Install: " + c)
 
     if option:
-        show_app(option["app"])
+        show_app(option["app"],c)
     else:
-        return
+        show_categories()
 
-def show_app(a):
+def show_app(a,c):
     clear()
-    with WaitingMessage():
+    with WaitingMessage(title=title, text="Loading app description..."):
         app_info = store.get_app(a)
 
-    install = prompt_boolean(app_info["description"], title=a, true_text="Install", false_text="Back")
+    app_text = "App:\n"
+    try:
+        app_text += app_info["title"] + '\n'
+    except:
+        pass
+    finally:
+        app_text += a + '\n\nDescription:\n' + app_info["description"].strip()
```

Comment:
This pattern of building strings gets the job done, but a nicer way would be to use Python string formatting, for example:

```python
name = app_info.get("title", a) # Try to get the 'title' key from app_info, falling back to the value of a if not present
desc = app_info["description"].strip()
app_text = """App:\n{}\n\nDescription:\n{}""".format(name, desc)
```